examples/data-objects/monaco/src/view.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { MonacoRunner } from "./chaincode";
+
+export const MonacoRunnerView = (elm: HTMLElement, model: MonacoRunner) => {
```

Comment:
This feels a bit like a hybrid of a React function component and a ReactDOM.render call.  If you want to feel closer to a function component, maybe consider dropping the `elm` arg and instead returning the subtree root element such that the caller can attach it to the `elm` themselves.

Reply:
Alternatively, could promote this to be a full `IFluidMountableView` class to satisfy the other comment.